File: Basic/prime.java
import java.util.*;

public class prime
 {
         public static void main(String[] args)
        {
           Scanner scn=new Scanner(System.in);
           System.out.println("enter a number");
           int n;
           n=scn.nextInt();
           for(int i=1;i<=n;i++)
           {
               System.out.println("enter number "+i);
               int x;
               x=scn.nextInt();
               boolean flag=true;
               for(int j=2;j<=x/2;j++)
               {
                   if (x%j==0)
                   {
                       flag=false;
                       break;
                   }
               }
               if (flag==true)
               {
                   System.out.println("prime");

               }
               else
               {
                   System.out.println("composite");
               }
               
           }
        }
    
}

File: Basic/prime.cpp
#include<iostream>

using namespace std;

int main(int argc,char** argv)
{
    int n;
    cout<<"enter the number";
    cin>>n;
    bool flag=true;
    for( int i=2; i*i<=n; i++)
    {
        if(n%i==0)
        {
            flag=false;
            break;
        }
    }
    if (flag==true)
    {
        cout<<"prime";
    }
    else 
    {
        cout<<"composite";
    }
}